components/functions-library-mgt/org.wso2.carbon.identity.functions.library.mgt/src/test/java/org/wso2/carbon/identity/functions/library/mgt/FunctionLibraryManagementServiceTest.java

```diff
@@ -123,28 +123,28 @@ public void createFunctionLibrary(Object functionLibrary, String tenantDomain) {
 
             assertEquals(functionLibraryManagementService.getFunctionLibrary(
                     ((FunctionLibrary) functionLibrary).getFunctionLibraryName(), tenantDomain)
-                                                         .getFunctionLibraryName(),
+                            .getFunctionLibraryName(),
                     ((FunctionLibrary) functionLibrary).getFunctionLibraryName());
 
             // Clean after test
             functionLibraryManagementService.deleteFunctionLibrary(
                     ((FunctionLibrary) functionLibrary).getFunctionLibraryName(), tenantDomain);
         } catch (FunctionLibraryManagementException e) {
```

Comment:
add a fail() before the catch, if we expect delete call to throw an exception

Reply:
[Fixed commit](https://github.com/wso2/carbon-identity-framework/pull/2679/commits/8c5deb0813742b0687efcb1daedf5123e568cb42)